libdevcrypto/AES.cpp

```diff
@@ -28,9 +28,264 @@
 
 using namespace std;
 using namespace dev;
-using namespace dev::crypto;
 using namespace CryptoPP;
 
+namespace
+{
+
+// This code is modified version of https://github.com/kokke/tiny-AES128-C.
```

Comment:
Please mention that it was released under the "unlicense", roughly equivalent to a release to the public domain.